java/test/jmri/jmrit/display/layoutEditor/load/Decorations-4-19-6.xml

```diff
@@ -1,320 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was this file deliberate deleted? I'm fine with it, just checking to be sure everything is OK.

Reply:
Oops.  Thanks for catching that! Will restore.